.travis.yml

```diff
@@ -24,6 +24,7 @@ env:
   matrix:
     - TEST="clang-format catkin_lint"
     - ROS_DISTRO=melodic
+    - ROS_DISTRO=kinetic
```

Comment:
So bullet works on kinetic / 16.04 now??

Reply:
Ah I see... it means Bullet gets disabled for kinetic, so the build will pass

Reply:
Bullet gets disabled if the version is below `2.87` which is the case for Xenial (I don't think the ROS distro has anything to do with it).